test/reviewController.test.js

```diff
@@ -124,4 +124,95 @@ describe('ReviewController API Tests', () => {
       expect(response.body.allProductReviews).toEqual([]);
     }
   });
+  //create a review
+  it('should create a review successfully', async () => {
+    await Review.deleteMany({});
+    const userCredentials = {
+      email: 'Emily@google.com',
+      password: 'secret',
+    };
+
+    const signedCookie = await loginAndReturnCookie(userCredentials);
+
+    const album = await Album.findOne({ albumName: 'Unique Album' });
+
+    const newReviewData = {
+      rating: 4,
+      title: 'Loved this review',
+      comment: 'Recommend it"',
+      user: user._id,
+      album: album._id,
+    };
+    const response = await request(app)
+      .post(`/api/v1/reviews/album/${album._id}`)
+      .set('Cookie', signedCookie)
+      .send(newReviewData);
+
+    expect(response.status).toBe(StatusCodes.CREATED);
+    expect(response.body).toHaveProperty('review');
+    // If no album was found
+    if (!album) {
+      expect(response.status).toBe(StatusCodes.NOT_FOUND);
+    }
+    //create second review for the same album
+    const secondReviewData = {
+      rating: 4,
+      title: 'Loved this review',
+      comment: 'Recommend it"',
+      user: user._id,
+      album: album._id,
+    };
+    const res = await request(app)
+      .post(`/api/v1/reviews/album/${album._id}`)
+      .set('Cookie', signedCookie)
+      .send(secondReviewData);
+    // this user already submitted a review for this album
+    expect(res.status).toBe(StatusCodes.CONFLICT);
+  });
+
+  //  update a review
+  it('should update a review successfully', async () => {
+    const userCredentials = {
+      email: 'Emily@google.com',
+      password: 'secret',
+    };
```

Comment:
Since we use this in two separate tests; it may be worthwhile to move this to an upper scope so that we can refer to the same variable in both tests. That way if the password or email ever has to be changed (eg: let's say in the future we decide we only want to allow passwords longer than 6 characters), then there's just one place to look to make that change, instead of having to update multiple tests.